src/main/java/lotto/domain/Lotto.java

```diff
@@ -0,0 +1,41 @@
+package lotto.domain;
+
+import java.util.List;
+import lotto.util.ErrorMessage;
+
+public class Lotto {
+    public static final int ONE_LOTTO_PRICE  = 1000;
+    private final List<Integer> numbers;
+
+    public Lotto(List<Integer> numbers) throws IllegalArgumentException {
+        validate(numbers);
+        this.numbers = numbers;
+    }
+
+    private void validate(List<Integer> numbers) {
+        if (numbers.size() != 6) {
+            throw new IllegalArgumentException(ErrorMessage.NEED_6_NUMBERS);
+        }
+        for (int index = 0; index < numbers.size(); index++) {
+            if (isOutrange(numbers.get(index))) {
+                throw new IllegalArgumentException(ErrorMessage.INVALID_LOTTO_NUMBER);
+            }
+            if (index >= 1 && isDuplicate(numbers, index)) {
+                throw new IllegalArgumentException(ErrorMessage.SAME_LOTTO_NUMBER);
+            }
+        }
+    }
+
+    private boolean isOutrange(int number) {
+        return !(1 <= number && number <= 45);
```

Comment:
```suggestion
        return number < 1 || number > 45;
```
`isOutrange()`라는 메소드명에 맞게 조건문의 부정을 뒤집으면 조금 더 로직이 잘 보이지 않을까 싶습니다!

Reply:
왓 suggestion 이런 기능도 있군요 조언과 더불어 하나 배워갑니다

Reply:
저는 여기에 호출부(Call Site)와 정의부(Definition Part)라는 관점을 조금 더해서 개인적인 의견을 말씀드리고 싶습니다.

이 메서드를 사용하는 호출부에서 if (!isOutrange(number)) 같은 이중 부정 코드가 나오는 것은 피하는 게 좋다고 생각합니다.

지금 코드는 메서드 정의부(코드를 구현한 곳) 입니다

작성해주신 원본 코드 return !(1 <= number && number <= 45);가 가진 장점은,
'1에서 45' 라는 핵심 범위를 직관적으로 보여준다고 생각했습니다.

결국 이 코드 또한 "이 범위(1~45)에 속하지 않는다(!)"는 의미로, 그 의도가 명확하게 읽힌다고 생각합니다

@leego0221 님이 말씀해주신 number < 1 || number > (경계값 검사) 방법도 메서드 명에 맞고 명확하다고 생각합니다.
다만 (1 <= number <= 45) (범위 검사)도 '범위 밖'이라는 개념을 잘 표현한 선택이라고 생각합니다~